### PR TITLE
Add sex to the engine.

### PR DESCRIPTION
**Pull request by IntangibleMatter**
Adds sex to the engine.

### CODE REVIEW

File: main/main.cpp [hunk 1]
```diff
@@ -3742,6 +3742,11 @@ void Main::force_redraw() {
 	force_redraw_requested = true;
 }
 
+// add sex
```
Comment (YeldhamDev) [inline main/main.cpp:3745]:
```suggestion
// Add sex.
```

**Review by YeldhamDev — CHANGES_REQUESTED** [review]

**Comment by Beeper-Skeeper** [comment]
now THIS is an "!eggrp" moment

**Comment by ToasterBiome** [comment]
now THIS is an "!eggrp" moment

**Comment by YuriSizov** [comment]
Hey there! Appreciate the meme's trending potential for any project, I really do. We generally don't mind a joke on this issue tracker from time to time. But please avoid referring to adult or risque themes since not everyone would be comfortable with such jokes, and this is, first and foremost, a workspace for hundreds of people. In other words, please treat this as a place of work, and if something is NSFW then it's not safe for this place as well.

And as a sidenote, if you want to create some humorous content on GitHub, don't do it in pull requests. People keep close attention to pull requests and will be annoyed by something like this, and you also waste CI time on this.

Cheers.

**Pull request closed without merging**